File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Tam giác sao rỗng/NGOISAO6.cpp
#include<bits/stdc++.h>
using namespace std;
int main() 
{
    int h;
    cin >> h;
    // In tam giác cân sao rỗng
    for (int i = 0; i < h; ++i) 
    {
        for (int j = 0; j < 2 * h - 1; ++j) 
        {
            if (i == h - 1 || i + j == h - 1 || j - i == h - 1)
                cout << "*";
            else 
                cout << " ";
        }
        if (i < h - 1) 
            cout << endl;
    }
    return 0;
}

File: Upcoder/Lập trình cơ bản/3.Vòng lặp/Tam giác sao rỗng/NGOISAO6.py
# Nhập chiều cao của tam giác
h = int(input())

# In tam giác cân sao rỗng
for i in range(h):
    for j in range(2 * h - 1):
        if i == h - 1 or i + j == h - 1 or j - i == h - 1:
            print("*", end="")
        else:
            print(" ", end="")
    if i < h - 1:
        print()
